content/tutorials/wallets.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Wallet Basics
+description: 
+---
+
+Ocean users need a wallet to manage their ETH and OCEAN tokens. 
+
+## Recommendations
+
+- **Easiest:** Use MetaMask. See the tutorial about [how to set up MetaMask](/tutorials/metamask-setup/). OR
+- **Still easy, but more secure:** Get a Trezor or Ledger hardware wallet, and use MetaMask to interact with it.
+
```

Comment:
link here also to our official list of wallets: https://oceanprotocol.com/token#get

Reply:
done